hack/pin-bundle-images.sh

```diff
@@ -33,9 +33,11 @@ for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path |
     GITHUB_USER=$(echo $MOD_PATH | sed -e 's|github.com/\(.*\)/.*-operator/.*$|\1|')
     REPO_CURL_URL="https://quay.io/api/v1/repository/openstack-k8s-operators"
     REPO_URL="quay.io/openstack-k8s-operators"
+    CURL_REGISTRY="quay.io"
```

Comment:
what about using this var also in REPO_CURL_URL and REPO_URL?

Reply:
done